### ISSUE TITLE
Error when OCR after upgrade to 7.4.1

### ISSUE THREAD

**Issue by ostasevych** [cyanfish/naps2#356]
Hi! After upgrading from 7.4.0 to 7.4.1 I am constantly getting error when performing OCR job:
```
System.FormatException: Input string was not in a correct format.
   at float Number.ParseSingle(string value, NumberStyles options, NumberFormatInfo numfmt)
   at (float m, float b) NAPS2.Ocr.TesseractOcrEngine.GetBaselineParams(XElement element)
   at OcrResult NAPS2.Ocr.TesseractOcrEngine.CreateOcrResult(XDocument hocrDocument)
   at async Task<OcrResult> NAPS2.Ocr.TesseractOcrEngine.ProcessImage(ScanningContext scanningContext, string imagePath, OcrParams ocrParams, CancellationToken cancelToken)
```

The OCR job is not done at all.
Also, I have checked OCR works fine when downgraded to v7.4.0.

Win10 x64
NAPS2 7.4.1

**Comment by ostasevych** [comment]
> Do you have a sample (non-ocr'd) PDF you can attach with the problem? What OCR language are you using?

Any file which I try to OCR after scanning or after importing pdf file. I tried Ukrainian and Ukrainian+English.

**Comment by cyanfish** [comment]
I can't reproduce the problem, so a sample would be helpful.

**Comment by zisco67** [comment]
Same here - used language is German; downgrade solves the problem!

**Comment by cyanfish** [comment]
@zisco67 Do you have a sample (non-ocr'd) PDF you can attach with the problem?

I can't help if I don't know what the problem is.

**Comment by cyanfish** [comment]
This should be fixed in [7.4.2](https://www.naps2.com/download).

**Comment by ostasevych** [comment]
Hi! Thanks, it is fixed.
